### DISCUSSION TITLE
[Appium Flutter driver] Is it possible to confirm if the slider option is activated or not?

### DISCUSSION THREAD

**Discussion by Harsha-Gelato** [appium/appium#17344] [General]
Hi All,

I am automating the testcase using appium flutter driver in which when I will click on the each notification, it will disable the slider and I would like to confirm if the slider is in active state or not? 

Any ideas how can I achieve this?
![Simulator Screen Shot - iPhone 13 - 2022-08-09 at 12 56 23](https://user-images.githubusercontent.com/103107117/183641739-15d26fdc-1e24-45c6-8149-35a8a64794ee.png)
?

**Comment by jlipps** [comment]
given that none of the appium team maintains the flutter driver, you may get a better response by posing the question at the flutter driver repo?

**Reply by Harsha-Gelato** [reply]
Unfortunately I don't see "**Discussion**" options for https://github.com/appium-userland/appium-flutter-driver
<img width="940" alt="Screenshot 2022-08-09 at 17 19 53" src="https://user-images.githubusercontent.com/103107117/183705090-eaf0e892-077b-4bfe-8445-4105c99a2b46.png">

**Reply by jlipps** [reply]
fair enough. i mean it's fine to have it here. i don't have any good ideas though.

**Reply by Harsha-Gelato** [reply]
If I have any questions related to appium flutter driver, where should I post those questions? Any idea? I usually post here or https://discuss.appium.io/ but when my question is related to appium flutter driver, unfortunately I don't get any answer :(

**Comment by KazuCocoa** [comment]
Can you get the slider value via `driver.execute('flutter: getRenderTree')` command? https://github.com/appium-userland/appium-flutter-driver
I wonder if some finders and getText etc can get the value.

**Reply by Harsha-Gelato** [reply]
I resolved this issue, by adding unique keys in the dev code for slider enabled and disabled option as it was missing at dev end. So, this ticket can be resolved now.
